setup.py

```diff
@@ -50,9 +53,9 @@
 setup(
     name="djangocms-frontend",
     version=__version__,
-    author="fsbraun",
```

Comment:
You could keep your name on it and add the maintained attrs for the dCA. You did write it after all.